environments/all/firstrun

```diff
@@ -6,6 +6,12 @@ set -euo pipefail
 # settings we’re about to change
 osascript -e 'tell application "System Preferences" to quit'
 
+###############################################################################
+# One-off Dependencies                                                        #
+###############################################################################
+
```

Comment:
This runs after plugins, so we just need Rosetta to launch or is it needed at install time?

Reply:
ah. right. yea, this won't work here.